src/multi_vector_simulator/utils/__init__.py

```diff
@@ -109,10 +109,16 @@ def compare_input_parameters_with_reference(folder_path, ext=JSON_EXT):
     under the key {EXTRA_PARAMETERS_KEY}
     """
     if ext == JSON_EXT:
-        # load the mvs input json file into a dict
-        json_file_path = os.path.join(folder_path, JSON_FNAME)
-        with open(json_file_path) as fp:
-            main_parameters = json.load(fp)
+
+        if isinstance(folder_path, dict):
+            # the folder_path argument is already a dict
+            main_parameters = folder_path
+        else:
+            # load the mvs input json file into a dict
+            json_file_path = os.path.join(folder_path, JSON_FNAME)
+            with open(json_file_path) as fp:
+                main_parameters = json.load(fp)
```

Comment:
Does this mean that we now can give the parsed json file directly as an argument for the MVS? In that case, what happens if `-ext csv` or `-ext json` is given as an argument as well?

Reply:
No, this means that you can provide the parse json to the function `compare_input_parameters_with_reference`

You can already give the parsed json file direcly as an argument of the MVS if you use the `server.run_simulation` instead of `cli.main` (this is what gets executed when running `mvs_tool`). I was not planning to allow already parsed json to command line.